frontend/src/components/clinics/clinics.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import Clinic from './components/clinic/clinic';
+import DoctorsFiltration from '../doctors-filtration-panel/doctors-filtration-panel';
```

Comment:
```suggestion
import { Filtration } from './components';
```
Also rename component/file and etc